docs/step.md

```diff
@@ -46,6 +46,10 @@ Check [react-floater](https://github.com/gilbarbara/react-floater) for more info
 **placementBeacon** {string} ▶︎ placement  
 The placement of the beacon. It will use the placement if nothing is passed and it can be: `top, bottom, left, right`.
 
+**spotlightTarget** {Element\|string} ▶︎ target
```

Comment:
There's no default value for this prop. Please remove the "▶︎ target"

Reply:
Actually, as you can see in Overlay.js value from `target` is a default value.
So when you don't define `spotlightTarget` it will work as it is now and tooltip target will be spotlighted.
If the user wants to spotlight another element he should define `spotlightTarget` prop.

Reply:
Overlay value "from" target? There's no default value for the target prop.
It is required for the step to be valid.